modules/3_helpernode/helpernode.tf

```diff
@@ -101,6 +101,55 @@ resource "null_resource" "config" {
             "cd ocp4-helpernode && git checkout ${var.helpernode_tag}"
         ]
     }
+}
+
+resource "null_resource" "prep_helpernode_tools_curl" {
+    triggers = {
+        bootstrap_count = var.bootstrap_port_ip == "" ? 0 : 1
+    }
+    count = length(regexall("\\.zip$", var.helpernode_repo)) > 0 ? 1 : 0
+
+    connection {
+        type        = "ssh"
+        user        = var.rhel_username
+        host        = var.bastion_ip[0]
+        private_key = var.private_key
+        agent       = var.ssh_agent
+        timeout     = "${var.connection_timeout}m"
+        bastion_host = var.jump_host
+    }
+
+    provisioner "remote-exec" {
+        inline = [
+            "mkdir -p .openshift",
+            "rm -rf ocp4-helpernode",
```

Comment:
Just to be safe delete any ocp4-helpernode.zip as well.

Reply:
Done